### PR TITLE
Webnav

### PR DESCRIPTION
**Pull request by John-Arboleda**
## Complete website with navigation

Awesome Books is a basic website that allows users to add/remove books from a list. This is achieved by using JavaScript objects and arrays. The DOM is modified dynamically and adds basic events.

In this pull request, the project is refactored to add three new sections (Contact info section, a Navigation bar, and a Footer), on top of the two sections it has already (Books list and Add book form).

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -73,3 +101,19 @@ window.onload = () => {
...
+  const options = {
+    weekday: 'long',
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+  };
+  now.innerHTML = new Intl.DateTimeFormat('en-US', options).format(date);
+}
```
Comment (helman101) [inline index.js]: - [x] Please remember that you are supposed to use the `Luxon` library to get and display the current date, here you are using `Date.now()` to get the date, kindly fix this 😄

File: style.css [hunk 2]
```diff
@@ -1,9 +1,52 @@
...
   margin: auto;
   width: 400px;
   text-align: center;
 }
 
+.navigation {
+  display: flex;
+  width: 800px;
+  justify-content: space-between;
+}
```
Comment (helman101) [inline style.css]: - [ ] Please make your project look as similar as possible to the given template. 😄
    - This is your project:
      imagen
    - This is the given template:
      imagen

**Review by helman101 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🔴 

Hi @John-Arboleda & @franklinben23  👋, 

Nice job so far with this project 😄, but there are still some changes to do.

## Required Changes ♻️

- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by descholar-ceo — APPROVED** [review]
Hi @franklinben23 and @John-Arboleda , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**